packages/SwingSet/docs/liveslots.md

```diff
@@ -4,8 +4,6 @@ This runtime treats vats as if they were userspace processes in an operating sys
 
 "Liveslots" is a particular dispatch mechanism that uses Maps/WeakMaps to track "live objects". Imported references to objects on other vats are represented by special "Presence" objects. Liveslots implements the `E` wrapper which allows messages to be sent through presences: `promise = E(presence).methodname(args)`. Presences can be sent in arguments or returned from method calls.
 
-In the future, we expect to have a syntax desugaring phase, so `E(p).foo(arg)` can be written as `p~.foo(arg)`. This "tildot notation" indicates that we're effectively calling `p.foo(arg)`, but 1: it always happens in a later turn (and always returns a Promise), and 2: `p` can refer to something on a remote host.
```

Comment:
We don't even expect to have this in the future?

Reply:
I think we *are* expecting to bring tildot back some day, but it didn't seem useful to mention that hope in these docs, nor to explain the eventual eventual-send syntax. I think I left one note about future plans somewhere (now I forget where), and that seemed like enough.